hopsworks-api/src/main/java/io/hops/hopsworks/api/tensorflow/TensorboardProxyServlet.java

```diff
@@ -207,7 +207,7 @@ protected void service(HttpServletRequest servletRequest,
         targetUriObj = new URI(targetUri);
         targetUriHost = new URI(theHost);
       } catch (Exception e) {
-        throw new ServletException("Trying to process targetUri init parameter: ", e);
+        return;
```

Comment:
@robzor92 add some logging here. If it's not an error that would indicate a problem with the backend, we can set it in FINE level.

Reply:
done